widget/install.md

```diff
@@ -0,0 +1,65 @@
+---
+description: "Install the widget page"
+---
+
+# Install the Bando Widget
+
+To install the Bando Widget, you can follow these steps:
+
+{% tabs %}
+
+{% tab title="npm" %}
+
+```bash
+npm install @bando/widget
+```
```

Comment:
almost same but the actual npm org is named bandohq because bando was already taken.